config/apparmor/abstractions/start-container

```diff
@@ -41,3 +41,4 @@
 
   change_profile -> lxc-*,
   change_profile -> unconfined,
+  change_profile -> :lxc-*:unconfined,
```

Comment:
According to http://wiki.apparmor.net/index.php/AppArmorNamespaces#name, the `//` is optional.
In my use case, `unconfined` provided a satisfying result. An empty namespace was enough to start the container and make it believe it was unconfined. An initial profile wasn't needed. 
Using `*` would be ok too.